app/scripts/lib/constants.js

```diff
@@ -5,6 +5,7 @@
 define(function (require, exports, module) {
   'use strict';
 
+  var Duration = require('duration');
```

Comment:
@larissagaulia looking at the tests failures I see that `app/scripts/lib/constants.js` is also used by server code, which means we won't be able to use Duration here. So you need to undo this change.